client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -173,8 +173,12 @@ class SharingServiceDescription extends Component {
 				args: { service: this.props.service.label },
 				context: 'Sharing: Publicize',
 			} );
-		} else if ( 'function' === typeof this.props.descriptions[ this.props.service.ID ] ) {
-			description = this.props.descriptions[ this.props.service.ID ].call( this );
```

Comment:
From an ignorance standpoint: we are replacing this condition because it was only relevant for the old `instagram` service, and is now unnecessary with `instagram_basic_display`?

Reply:
It's still necessary, the issue is that for the new Instagram service the `service.ID` is kebab case (`instagram-basic-display`) and we need to change it to snake case in order to be a valid function name. Previously this wasn't necessary as we've changed the service ID to be snake case, like `google_photos`. I've decided not to do that here as the standard for Keyring is kebab case, and it would be good to be able to push this service upstream to the Keyring plugin.